Index: host.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/host.py b/host.py
new file mode 100644
--- /dev/null	(date 1626105532268)
+++ b/host.py	(date 1626105532268)
@@ -0,0 +1,37 @@
+import socket
+import threading
+from netifaces import interfaces, ifaddresses, AF_INET
+
+class Host:
+    @staticmethod
+    def _getip():
+        for ifaceName in interfaces():
+            addresses = [i['addr'] for i in ifaddresses(ifaceName).setdefault(AF_INET, [{'addr': ''}])]
+            for a in addresses:
+                if a != '' and a != "127.0.0.1":
+                    return a
+
+    def __init__(self, port, localhost):
+        self.port = port
+        self.ip = self._getip()
+        if localhost: self.ip = '127.0.0.1'
+        self.conns = []
+
+    def manage_conn(self, conn, addr):
+        print(f"Connected {conn}, {addr}")
+        connected = True
+        while connected:
+            pass
+
+    def start(self):
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+            s.bind((self.ip, self.port))
+            s.listen()
+            conn, addr = s.accept()
+            x = threading.Thread(target=self.manage_conn)
+            x.start()
+
+
+if __name__ == '__main__':
+    host = Host(2000, True)
+    host.start()
\ No newline at end of file
Index: restart.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/restart.sh b/restart.sh
new file mode 100755
--- /dev/null	(date 1626105825167)
+++ b/restart.sh	(date 1626105825167)
@@ -0,0 +1,4 @@
+echo "Restarting port " $1
+
+ufw deny $1
+ufw allow $1
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1626099445591)
+++ b/.idea/vcs.xml	(date 1626099445591)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/discord.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/discord.xml b/.idea/discord.xml
new file mode 100644
--- /dev/null	(date 1626099445574)
+++ b/.idea/discord.xml	(date 1626099445574)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DiscordProjectSettings">
+    <option name="show" value="PROJECT_FILES" />
+  </component>
+</project>
\ No newline at end of file
